integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorWlsUpgrade.java

```diff
@@ -322,16 +331,18 @@ private void scaleClusterUpAndDown() {
         "Could not get the Operator external service node port");
     logger.info("externalRestHttpsPort {0}", externalRestHttpsPort);
 
-    // check domain can be managed from the operator by scaling the cluster
-    scaleAndVerifyCluster("cluster-1", domainUid, domainNamespace,
-        managedServerPodNamePrefix, replicaCount, 3,
-        true, externalRestHttpsPort, opNamespace, opServiceAccount,
-        false, "", "", 0, "", "", null, null);
+    String clusterName = domainUid + "-" + "cluster-1";
+    logger.info("Updating the cluster {0} replica count to 3", clusterName);
+    boolean p1Success = scaleCluster(clusterName, domainNamespace,3);
+    assertTrue(p1Success,
+        String.format("Patching replica to 3 failed for cluster %s in namespace %s",
+            clusterName, domainNamespace));
 
-    scaleAndVerifyCluster("cluster-1", domainUid, domainNamespace,
-        managedServerPodNamePrefix, replicaCount, 2,
-        true, externalRestHttpsPort, opNamespace, opServiceAccount,
-        false, "", "", 0, "", "", null, null);
+    logger.info("Updating the cluster {0} replica count to 2");
+    p1Success = scaleCluster(clusterName, domainNamespace,2);
+    assertTrue(p1Success,
+        String.format("Patching replica to 1 failed for cluster %s in namespace %s",
```

Comment:
incorrect replica count in error message

Reply:
Done